### ISSUE TITLE
[Bug]: Cant start seamless co-op

### ISSUE THREAD

**Issue by SupRocket492** [LukeYui/EldenRingSeamlessCoopRelease#576]
### Discord Username

soupRerouket#4687

### Describe the issue in detail

When I start the application it turns the screen white, then black then closes seamless co-op

### Steps to recreate the bug

I played this last night and it was working perfectly fine but now it wont work and im not sure why

### Version

1.5.0 (Latest)

### Relevant Crash log

```shell
Faulting application name: eldenring.exe, version: 1.9.0.0, time stamp: 0x6405b0f3
Faulting module name: elden_ring_seamless_coop.dll, version: 0.0.0.0, time stamp: 0x63c45f5b
Exception code: 0xc0000005
Fault offset: 0x000000000001ce39
Faulting process id: 0x0x1860
Faulting application start time: 0x0x1D95D724534ACC8
Faulting application path: c:\program files (x86)\steam\steamapps\common\ELDEN RING\Game\eldenring.exe
Faulting module path: c:\program files (x86)\steam\steamapps\common\ELDEN RING\Game\mods\elden_ring_seamless_coop.dll
Report Id: 026e6196-c5e8-4830-b476-e811f313a90f
Faulting package full name: 
Faulting package-relative application ID:
```

**Comment by SupRocket492** [comment]
I see now that the game updated, so now I have my answer on why it doesn't work

**Comment by bentrooper** [comment]
Yes, same thing here, game updated this morning

**Comment by marioncalpe** [comment]
i have a same bug and i don't know fix this. anyone help ?

**Comment by KRST904** [comment]
https://www.makeuseof.com/how-to-downgrade-steam-games/

Quite Bothersome but i think it's one of the few ways

**Comment by jinhtz** [comment]
> > i have a same bug and i don't know fix this. anyone help ?
> 
> You need to wait until seamless co-op is updated. When seamless co-op is updated update the mod then verify integrity of files in steam

Thank you for verifying.  I assumed this would be the case.

**Comment by rysiuwroc** [comment]
> https://www.makeuseof.com/how-to-downgrade-steam-games/
> 
> Quite Bothersome but i think it's one of the few ways

TLDR:

1. Close steam
2. win+r -> steam://open/console
3. go to CONSOLE tab
4. paste: download_depot 1245620 1245621 1372657426714032507 
5. wait for finish, it may take some time since its 40+gigs
6. paste:  download_depot 1245620 1245624 1790146326279181276
7. this one is fast, 23megs
8. copy all the downloaded content from 2 folders and hard paste to elden ring folder
9. profit

This worked for me

**Comment by twimerig** [comment]
Yes, mod is not working after update. Same white then black screen.
Please update mod! I am playing with my wife and enjoying it so much.
Best 2023 year entertainment

**Comment by LukeYui** [comment]
1.5.0 is not compatible with game version 1.09

Thanks.
